octoprint_discordremote/command.py

```diff
@@ -330,6 +334,29 @@ def status(self):
 
                 builder.add_field(title='Time Spent', text=self.plugin.get_print_time_spent(), inline=True)
                 builder.add_field(title='Time Remaining', text=self.plugin.get_print_time_remaining(), inline=True)
+        #get throttled for pi
+        sb2 = subprocess.Popen(['vcgencmd', 'get_throttled'], stdout=subprocess.PIPE)
```

Comment:
what happens if vcgencmd isnt on the system, people do run octoprint on non-rasppi systems.

Reply:
That's the issue(this is the issue that comes up on circleci). If there is some way that I can tell if it is run on a raspberry pi or not(Some sort of query?), then that would be the solution. Otherwise we could add an option in the GUI(default would be off) to turn on Pi support. It's probably also possible to see if "Pi Support Plugin"(the plugin installed as default in OctoPi after a recent version) is installed and use that as the boolean. This wouldn't work for people  who installed it manually on raspbian or something similar, so some combination,if possible, of all these options is definitely best.

Reply:
if vcgencmd doesnt exist, subprocess will throw FileNotFoundException, just wrap your code in a try catch, and ignore the error if it is raised.

Reply:
ok